rules/xcodeproj.bzl

```diff
@@ -115,18 +118,28 @@ def _xcodeproj_aspect_impl(target, ctx):
     else:
         srcs = []
         asset_srcs = []
+        other_cflags = []
         for attr in _dir(ctx.rule.files):
             if attr == "srcs":
                 srcs += getattr(ctx.rule.files, attr, [])
             else:
                 asset_srcs += getattr(ctx.rule.files, attr, [])
         srcs = [f for f in srcs if _is_current_project_file(f)]
         asset_srcs = [f for f in asset_srcs if _is_current_project_file(f)]
+        if CcInfo in target:
+          for fi in target[CcInfo].compilation_context.framework_includes.to_list():
```

Comment:
instead of getting this as a list, we should directly propogate this depset -- alternatively, we might want to just propogate a raw `compilation_context` inside the `_SrcsInfo`, so we can use the `cc_common` aparatus for merging the values. then, in the implementation of the xcodeproj rule, we can expand out the depsets as needed, and turn them into `OTHER_CFLAGS` there (or alternatively use the native xcconfigs, such as `FRAMEWORK_SEARCH_PATHS` and `GCC_PREPROCESSOR_DEFINITIONS`)